### ISSUE TITLE
Some of the commands should run as user not root. 

### ISSUE THREAD

**Issue by Co2333** [iabem97/topanga#51]
It comes out the problems bad cpu type in executable. It could run as user, but not root. 
![1f0a72f5-cbe4-490e-9304-d2b1def036a8](https://user-images.githubusercontent.com/25259084/34338225-761dca00-e9a1-11e7-9516-66f05ea3a640.png)
![c3629557-a0ae-4ef9-be9a-207eca64c177](https://user-images.githubusercontent.com/25259084/34338227-765b5244-e9a1-11e7-866e-c6649f2882e1.jpeg)

**Comment by idiidk** [comment]
Yeah, nearly all binaries that i try to run as root give this error. Not specific to setuid or gzip.

**Comment by Co2333** [comment]
@idiidk I tried to untar the bs2.tar and copy them from /bin /shin /usr/apt/ to cover the old one and then fixes some of the issues. But reboot not work.
